core/tests/protractor/collections.js

```diff
@@ -83,6 +81,32 @@ describe('Collections', function() {
     users.logout();
   });
 
+  it('interacts with the collection player as a guest', function() {
+    collectionPlayer.goToDemoCollection();
+    // Check that we are in the guest state (sign in button visible).
+    collectionPlayer.verifySignInButton();
+    // Verify that there are 5 uncompleted explorations (initial state).
+    collectionPlayer.verifyNumUncompletedExplorations(5);
+    // Play the third exploration in the demo collection.
+    collectionPlayer.playDemoExplorationThree();
+    // Verify that the suggested exploration is the next in the collection.
+    collectionPlayer.verifySuggestedExplorationAsGuest();
+    // Go back to collection player.
+    collectionPlayer.goToDemoCollection();
+    // Verify that there are 5 uncompleted explorations, since this is a guest.
+    collectionPlayer.verifyNumUncompletedExplorations(5);
+  });
+    
+  it('interacts with the collection player as a logged-in user', function() {
+    users.login('alice@collections.com');
+    collectionPlayer.goToDemoCollection();
+    // Verify that profile dropdown thumbnail appears, since we are logged in.
+    collectionPlayer.verifyProfileDropdown();
+    // Verify that there are 5 uncompleted explorations (initial state).
+    collectionPlayer.verifyNumUncompletedExplorations(5);
+    // This test (i.e. part 3) needs to be finished
```

Comment:
Format should be:

    // TODO(Ian91): ...

and the "..." should specify, in detail, exactly what needs to be done, so that someone with almost no context can pick it up and do it.